src/main/java/com/netgrif/application/engine/workflow/web/responsebodies/LocalisedTaskResource.java

```diff
@@ -36,7 +36,7 @@ private void buildLinks() {
         add(WebMvcLinkBuilder.linkTo(WebMvcLinkBuilder.methodOn(TaskController.class)
                 .getData(task.getStringId(), null)).withRel("data"));
         add(WebMvcLinkBuilder.linkTo(WebMvcLinkBuilder.methodOn(TaskController.class)
-                .setData(null, task.getStringId(), null)).withRel("data-edit"));
+                .setData(task.getStringId(), null, null)).withRel("data-edit"));
```

Comment:
is it correct?

Reply:
This is signature of method in question `public EntityModel<EventOutcomeWithMessage>  setData(Authentication auth, @PathVariable("id") String taskId, @RequestBody ObjectNode dataBody)` . So the order of the arguments is not correct.

Reply:
EntityModel<EventOutcomeWithMessage> setData(String taskId, ObjectNode dataBody, Locale locale)  its  okey...